### ISSUE TITLE
In the paper the dataset refers to VQAv2 validation sets and VQA-CP v2 test set?

### ISSUE THREAD

**Issue by alice-cool** [itaigat/perceptual-score#1]
Dear scholar, 
I want to ask whether you use the testing sets to evaluate the perception score. i.e. for VQA v2, the vqa models usually  use VQA v2 validation sets. For VQA-CP v2, the  vqa models usually  use VQA-CP v2 TEST sets.

![image](https://user-images.githubusercontent.com/16870890/171022696-74b9715b-ad68-4aed-a08c-25a83187af2b.png)

Your evaluation metric is on the two datasets (i.e.VQA v2 validation sets and  VQA-CP v2 TEST sets) to shuffle the datasets to be unpaired , and use your proposed formula to get the perception score.

**Comment by alice-cool** [comment]
The code is not complete.
![image](https://user-images.githubusercontent.com/16870890/171154395-7d1b5e8a-8e1b-4651-a019-ccebc58db944.png)
It can't show the formula in the paper.
The normalization term use the number of sample to denote...
Acc in formula(2) can not display.  Just use the loss function output to represent it. It is so unconvincing.

**Comment by itaigat** [comment]
Thanks for pointing that out, I've fixed it. This repository is a boilerplate for using our method. The loss term can be easily replaced with any other metric.
The original VQA code has been double-checked, and the results in the paper are accurate.
